modules/core/include/opencv2/core/hal/intrin_sse.hpp

```diff
@@ -1924,7 +1924,7 @@ OPENCV_HAL_IMPL_SSE_EXPAND(v_int32x4,  v_int64x2,   int,      _v128_cvtepi32_epi
 #define OPENCV_HAL_IMPL_SSE_EXPAND_Q(_Tpvec, _Tp, intrin)  \
     inline _Tpvec v_load_expand_q(const _Tp* ptr)          \
     {                                                      \
-        __m128i a = _mm_cvtsi32_si128(*(const int*)ptr);   \
+        __m128i a = _mm_loadu_si32(ptr);                   \
```

Comment:
> _mm_loadu_si32

This won't compile (GCC 5 / MSVS 2015 are in the support list).

Check approach from this PR with used sanitizer: #14022

Reply:
Done. Thanks for the example.
I was not able to reproduce the sanitizer error internally anymore (I guess some config changed) so I cannot check if this solution works for me.